pages/common/amass-enum.md

```diff
@@ -1,24 +1,28 @@
 # amass enum
 
 > Find subdomains of a domain.
-> More information: <https://github.com/OWASP/Amass/blob/master/doc/user_guide.md#the-enum-subcommand>.
+> More information: <https://github.com/owasp-amass/amass/blob/master/doc/user_guide.md#the-enum-subcommand>.
 
-- Passively find subdomains of a [d]omain:
+- Find (passively) subdomains of a [d]omain:
 
-`amass enum -passive -d {{domain_name}}`
+`amass enum -d {{domain_name}}`
 
 - Find subdomains of a [d]omain and actively verify them attempting to resolve the found subdomains:
 
 `amass enum -active -d {{domain_name}} -p {{80,443,8080}}`
 
-- Do a brute force search for subdomains:
+- Do a [brute] force search for sub[d]omains:
```

Comment:
```suggestion
- Do a brute force search for sub[d]omains:
```
Do we want to add a mnemonic for this whole this word? Well, the full, descriptive option would be `-brute-force`, but I don't know if we should highlight this whole word.